source/_components/recorder.markdown

```diff
@@ -102,6 +104,27 @@ If you only want to hide events from e.g. your history, take a look at the [`his
 
 Not all Python bindings for the chosen database engine can be installed directly. This section contains additional details which should help you to get it working.
 
+### {% linkable_title MariaDB %}
+
+For MariaDB you may have to install a few dependencies. On the python side we use the `mysqlclient`.
+
+```bash
+$ sudo apt-get install mariadb-server libmariadbclient-dev
```

Comment:
I assume that the you only need the server package if you want to run the database locally.

Reply:
That is true. Fixing.